functions/index.js

```diff
@@ -12,6 +13,32 @@ admin.initializeApp(functions.config().firebase);
 exports.createNewUser = functions.auth.user().onCreate(event => {
   const uid = event.data.uid;
   const email = event.data.email;
-  
+
   return admin.database().ref(`/users/${uid}`).set({ email });
 });
+
+exports.joinTheClub = functions.database.ref('/queues/joinTheClub/tasks/{taskId}').onWrite(event => {
+  if (event.data._newData) {
+    const { inviteCode, userId } = event.data._newData;
+
+    return admin.database().ref('bookClub').orderByChild('inviteCode').equalTo(inviteCode).once('value').then((response) => {
+      if (response.exists() && response.val()) {
+        const val = response.val();
+        const bookClubId = Object.keys(val)[0];
+        const bookClubName = Object.keys(val)[0].name;
+        const randomId = Math.random().toString(36).slice(-10);
```

Comment:
Nope! I'm hoping to make it a thing. At some point I'll do the front-end to actually display the notifications so the user has some indication that it worked (or that it failed miserably)